src/librustc/util/common.rs

```diff
@@ -244,7 +244,8 @@ fn get_resident() -> Option<usize> {
     use std::fs;
 
     let field = 1;
-    let contents = fs::read_string("/proc/self/statm").ok()?;
+    let contents = fs::read("/proc/self/statm").ok()?;
+    let contents = String::from_utf8(contents).ok()?;
```

Comment:
Why this change? Other than this r=me

Reply:
Using the new `read_to_string` function here would break bootstrapping from the previous release, where that name didn't exist.  This seemed simpler than doing some sort of conditional import based on cfg(stage0).

Reply:
Alright, sounds good.